2017-10/agenda.md

```diff
@@ -0,0 +1,46 @@
+# Node.js Collaborator Summit, 2017-10-06 - 2017-10-07, Vancouver, BC, Canada
+
+**This is a DRAFT proposed agenda**
+
+## 2017-10-06 Day One
+
+|    Time Slot    | Room |      Session     |        Description       |
+| --------------- | ---- | ---------------- | ------------------------ |
+|  9:00a - 11:00a | TBD  | Core Plenary     | All-Collaborator Plenary |
+| 11:00a - 12:00p | TBD  | TBD              | TBD                      |
```

Comment:
Would love to get the Moderation Team in a meeting/training session here. Input from the larger group as to what should happen moving forward for the team and its role in the project would be 👍

Reply:
Ok. So there could be two parts to that: A) A training *just* for the moderation team, which could take place during any of the Working Group Session times, and B) A discussion during the plenary about the team and it's role with the larger group.

Reply:
Code + Learn is happening on the 6th from 9AM until 12:30PM. I don't think the Collab Summit should start until 1:30PM on that day.

Reply:
2 hours for plenary seems about an 1 hour and 45 minutes too long. Plenary should be for planning smaller sessions. I don't think it's likely that anything productive will come out of trying to have a two hour conversation with everyone. If people have topics that they want broad input on, make a session for that topic.

Reply:
+1 on the Code-n-Learn, I wasn't sure what the timing was for that.
-1 on shortening the Plenary, I have a couple things in mind for that time that I'm going to be proposing shortly

Reply:
Yeah, sorry I didn't catch the times. We have Collab Summit room only for the second half of the day on Oct. 6 because Code + Learn is first half. My bad, y'all.